### ISSUE TITLE
Problem creating a DFP placement 

### ISSUE THREAD

**Issue by ialex** [googleads/googleads-python-lib#178]
im using 5.0.0

This SOAP request always returns 500 error:
```
name = "MY_PLACEMENT"
placement_srvc.createPlacements([{'name': name},])
```
SOAP
```
(b'<?xml version="1.0" encoding="UTF-8"?><SOAP-ENV:Envelope xmlns:ns1="http://s'
 b'chemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSche'
 b'ma-instance" xmlns:tns="https://www.google.com/apis/ads/publisher/v201608" x'
 b'mlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ns0="https:/'
 b'/www.google.com/apis/ads/publisher/v201608"><SOAP-ENV:Header><tns:RequestHea'
 b'der><tns:networkCode>917564123</tns:networkCode><tns:applicationName>O'
 b'rders (DfpApi-Python, googleads/5.0.0, Python/3.5.2)</tns:applicationName></'
 b'tns:RequestHeader></SOAP-ENV:Header><ns1:Body><ns0:createPlacements><ns0:pla'
 b'cements><ns0:name>MY_PLACEMENT</ns0:name></ns0:placements></ns0:'
 b'createPlacements></ns1:Body></SOAP-ENV:Envelope>')
```

**Comment by ialex** [comment]
4.8.0 returns the error properly that targetedAdUnitIds are required, i did try passing it empty array but still get 500 error on 5.0.0 im switching to 4.8.0 for now until it is fixed.

**Comment by Olmer1st** [comment]
Hi, i have the same issue with Error 500 with adwords client. 
Receiving error 500 instead real problem, for example instead OperationAccessDenied.ACTION_NOT_PERMITTED @ class com.google.ads.api.serviceimpls.campaignmgmt.criterion.campaigncriterion.M. When switched to 4.8.0 problem resolved.

**Comment by Olmer1st** [comment]
Additional AdWords report download failed with HTTP status code: 400

**Comment by thangduo** [comment]
Hi @ialex,

You are getting an HTTP 500 error because it is part of the SOAP v1.1 spec. Unlike REST, SOAP servers can return an HTTP 500 error a long with a detail Fault message for any problems that might have occurred on the server side while processing a request.

Can you please post the entire SOAP response, or the Fault message in the response?

Thanks

**Comment by ialex** [comment]
Hello @thangduo 

I wasn't able to see the SOAP response in the Django debug information but seems that

**Comment by thangduo** [comment]
We are aware of the HTTP 500 error's side effect and working on it.

In the mean time, you can still obtain the debugging information by either setting up an HTTP proxy (e.g. [Charles proxy](https://www.charlesproxy.com/), [Fiddler](http://www.telerik.com/fiddler), etc.) or moving the `try` statement from [line 102 ](https://github.com/googleads/googleads-python-lib/blob/master/googleads/util.py#L102) to anywhere before line 93.

**Comment by thangduo** [comment]
@ialex, are you able to see the cause of the HTTP 500 error?

**Comment by ialex** [comment]
@thangduo yes the error was caused because missing required parameter.

**Comment by thangduo** [comment]
We're putting in a fix so that the actual error is surfaced and not an HTTP 500 error for cases like this, so re-opening this issue to track that.

**Comment by thangduo** [comment]
Resolved in 5.1.0 release

**Comment by ialex** [comment]
@thangduo thanks for the update 👍
